horovod/common/optim/bayesian_optimization.cc

```diff
@@ -31,6 +32,16 @@ namespace common {
 
 const double NORM_PDF_C = std::sqrt(2 * M_PI);
 
+void GetSufficientStats(std::vector<double>& v, double* mu, double* sigma) {
+  double sum = std::accumulate(v.begin(), v.end(), 0.0);
+  *mu = sum / v.size();
+
+  std::vector<double> diff(v.size());
+  std::transform(v.begin(), v.end(), diff.begin(), std::bind2nd(std::minus<double>(), *mu));
```

Comment:
Does it take lambda instead of `std::bind2nd`?  Seems more reader-friendly :-)

Reply:
Good idea.  Done!